src/meshapi/tasks.py

```diff
@@ -12,21 +13,61 @@
 )
 from meshapi.views.panoramas import sync_github_panoramas
 from meshdb.celery import app as celery_app
+from meshdb.settings import DBBACKUP_STORAGE_OPTIONS
 
 
 @celery_app.task
-def run_database_backup() -> None:
-    logging.info("Running database backup task")
+def run_database_backup() -> bool:
+    # Don't run a backup unless it's prod1
+    environment = f"\"{os.environ.get('MESHDB_ENVIRONMENT', '')}\""
+    if os.environ.get("MESHDB_ENVIRONMENT") != "prod1":
+        logging.warn(f"Not running database backup. This environment is: {environment}")
+        return False
+
+    logging.info(f"Running database backup task. This environment is {environment}")
     if not os.environ.get("AWS_ACCESS_KEY_ID") or not os.environ.get("AWS_SECRET_ACCESS_KEY"):
         logging.error("Could not run backup. Missing AWS credentials!")
-        return
+        return False
 
     try:
         management.call_command("dbbackup")
     except Exception as e:
         logging.exception(e)
         raise e
 
+    return True
+
+
+@celery_app.task
+def reset_dev_database() -> bool:
+    # Only reset dev environments (very important!)
```

Comment:
This definitely isn't going to nuke my local setup if I run the celery worker, right?

Reply:
Not unless your environment matches `dev`.

Reply:
Ah I see, you edited the template to add a default of `local`